### PR TITLE
select companion data for end card

### PR DESCRIPTION
**Pull request by solitaa**

### CODE REVIEW

File: js/dist/rmp-vast.js [hunk 1]
```diff
@@ -746,6 +747,20 @@ COMPANION.parse = function (companionAds) {
...
+
+      var staticResourceUrl = _fw.default.getNodeValue(staticResource[0], true);
+
+      if (staticResourceUrl === null) {
+        continue;
+      }
+
+      this.companionForEndCard.push({
+        width: width ? width : 0,
+        height: height ? height : 0,
```
Comment (aleksanyan) [inline js/dist/rmp-vast.js:760]: I would do:
```
width: width || 0,
height: height || 0,
```

File: js/src/creatives/companion.js [hunk 2]
```diff
@@ -38,15 +40,27 @@ COMPANION.parse = function (companionAds) {
...
+
+      const staticResourceUrl = FW.getNodeValue(staticResource[0], true);
+      if (staticResourceUrl === null) {
+        continue;
+      }
+
+      this.companionForEndCard.push({
+        width: width ? width: 0,
+        height: height ? height: 0,
+        imageUrl: staticResourceUrl
```
Comment (aleksanyan) [inline js/src/creatives/companion.js:56]: looks to be a dublicate of ^ code, can we generalise & import?

**Review by aleksanyan — APPROVED** [review]
Due to the codebase state, I'm not being too picky 👍  so just a few comments.

File: js/src/creatives/companion.js (hunk 2 above)
Comment (solitaa) [reply js/src/creatives/companion.js:56]: This is not a duplicate. rmp-vast.js is generated after build.

**Pull request merged**